clip.py

```diff
@@ -99,6 +107,24 @@ def __init__(self, width, height):
         self.is_record = False
         self.outputsPorts = set()
         self.outputsPorts.add(Clip.DEFAULT_OUTPUT)
+        self.scenes = {}
```

Comment:
I think scenes must be ordered, so I would prefer a list of "scene"
Scene need to be ordered. First midi bindings will be "next scene" and "previous scene". So I think self.scenes should be a list of scene object. Each scene should have a name and a set/list of clip to start.